modules/android/vsandroid_vcxproj.lua

```diff
@@ -338,6 +338,17 @@
 				end
 			end
 
+			-- -fvisibility=<>
+			if cfg.visibility ~= nil then
+				local visibility_map = {
+					["Default"] 	= "default",
+					["Hidden"] 		= "hidden",
+					["Internal"] 	= "internal",
+					["Protected"] 	= "protected",
+				}
+				table.insert(opts, '-fvisibility=' .. visibility_map[cfg.visibility])
```

Comment:
This mapping exists in the [gcc tool](https://github.com/premake/premake-core/blob/master/src/tools/gcc.lua#L193). Does it need to be duplicated here?

Reply:
Can I access it like this? `gcc.cxxflags.visibility` ?

Reply:
I just tried it and I'm getting an error: `Error: [string "android/vsandroid_vcxproj.lua"]:268: attempt to index a nil value (global 'gcc')`

Reply:
I think it's better now https://github.com/ifarbod/premake-core/commit/044cddd81a00e65a82e4eada63100b3ff799e60c